docs/tracing/docs/010-details-jaeger.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Jaeger
+type: Details
+---
+
+## Overview
+[Jaeger](http://jaeger.readthedocs.io/en/latest/) is a monitoring and tracing tool for microservice-based distributed systems.
+It is used for monitoring and troubleshooting microservice-based distributed systems, including:
+
+- Distributed context propagation
+- Distributed transaction monitoring
+- Root cause analysis
+- Service dependency analysis
+- Performance and latency optimization
+
+## Usage 
+
+The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. Jaeger is compatible with the Zipkin protocol, which Istio and Envoy use to communicate with the tracing back-end. This allows you to use the Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
+
+For more details, see [Istio Distributed Tracing](https://istio.io/docs/tasks/telemetry/distributed-tracing.html).
+
+## Install Jaeger
+Jaeger installation is optional, and not available to use locally by default. You can install Jaeger on a Kyma instance and run it locally using Helm.
```

Comment:
we need to change it.

Jaeger is only optional for local installation and there only the steps are required. For the cluster, there are no installation actions needed to be taken. Perhaps the heading should also reflect same.

Reply:
I updated it, both the title and description. I mentioned the cluster installation (automatic) and local.